src/components/photo/Comment.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { useQuery, useQueryClient } from "react-query";
+import Proptypes from "prop-types";
+import dayjs from "dayjs";
+import axios from "axios";
+import styled, { ThemeProvider } from "styled-components";
+
+import {
+  ERROR_MESSAGE,
+  LOADING_MESSAGE,
+  RESPONSE_MESSAGE,
+} from "../../constants";
+import theme from "../../styles/theme";
+import Message from "../common/Message";
+
+const Comment = ({ comment }) => {
+  const [hasError, setHasError] = useState(false);
+  const [isDelete, setIsDelete] = useState(false);
+  const user = useSelector(state => state.user);
+  const photoIndex = useSelector(state => state.photo);
+  const index = photoIndex.currentIndex;
+  const { message, createdAt, createdBy } = comment;
+  const date = dayjs(createdAt).format("YY/MM/DD HH:mm");
+  const queryClient = useQueryClient();
+  const photos = queryClient.getQueryData("photos")?.data?.photos;
+  const photo = photos[index];
```

Comment:
개행을 해주면 가독성이 좋아질 것 같아요 !    
순서도 저는 훅부터 작성하는데 다들 어떻게 작성하시나용?? (밑에는 단순한 예시입니다!)
그리구 혹시... `  const date = dayjs(createdAt).format("YY/MM/DD HH:mm");` 이부분은 아예 상단에서 처리해주면 너무 멀까요?? 

```suggestion
const Comment = ({ comment }) => {
  const queryClient = useQueryClient();
  const photos = queryClient.getQueryData("photos")?.data?.photos;
  const photo = photos[index];
  
  const user = useSelector(state => state.user);
  const photoIndex = useSelector(state => state.photo);
  const index = photoIndex.currentIndex;
  
  const { message, createdAt, createdBy } = comment;
  
  const [hasError, setHasError] = useState(false);
  const [isDelete, setIsDelete] = useState(false);
  
  const date = dayjs(createdAt).format("YY/MM/DD HH:mm");
  
```